### PR TITLE
Fix library file

### PR DESCRIPTION
**Pull request by x3medima17**
Signed-off-by: Dumitru <dimasavva17@gmail.com>

### Description of the Change
Replaced Java library name with the right one

### Benefits
Working documentation 

### Possible Drawbacks 
None

### CODE REVIEW

**Comment by l4l** [comment]
It's not really true for all platforms, afaik

**Comment by neewy** [comment]
@l4l do we need this?

File: docs/source/guides/libraries/java.rst [hunk 1]
```diff
@@ -86,12 +86,12 @@ to the root folder of your project:
 
   cp dist/libirohajava.jnilib /path_to_your_project
 
-If you use **Gradle**, you need to copy ``example/java/dist/iroha_lib.jar`` to the
+If you use **Gradle**, you need to copy ``example/java/dist/iroha-bindings-example.jar`` to the
```
Comment (Alexey-N-Chernyshov) [inline docs/source/guides/libraries/java.rst]: I think you need to copy both `iroha-bindings-example.jar` and `libirohajava.jnilib`. And 'libirohajava.jnilib` in linux has extension `libirohajava.so`.

Comment (Alexey-N-Chernyshov) [reply docs/source/guides/libraries/java.rst]: Also please rename the lib on line 82.

File: docs/source/guides/libraries/java.rst [hunk 2]
```diff
@@ -86,12 +86,12 @@ to the root folder of your project:
...
 ``libs`` folder of your project
 
 .. code-block:: shell
 
-  cp dist/libirohajava.jnilib /path_to_your_project/libs
+  cp dist/iroha-bindings-example.jar /path_to_your_project/libs
```
Comment (Alexey-N-Chernyshov) [inline docs/source/guides/libraries/java.rst]: sap

File: docs/source/guides/libraries/java.rst (hunk 1 above)
Comment (l4l) [reply docs/source/guides/libraries/java.rst]: and `.dll` for windows i guess

**Comment by l4l** [comment]
Seems outdated

**Pull request closed without merging**